integration-tests/features/list-machines.feature

```diff
@@ -5,4 +5,5 @@ Scenario: Machine listing responsiveness
          | name       | definition          | ensure_fresh |
          | vm1        | centos6-guest-httpd | no           |
          | vm2        | centos7-target      | no           |
-   Then a full machine listing should take less than 120 seconds
+   Then a shallow machine listing should take less than 15 seconds
+    and a full machine listing should take less than 30 seconds
```

Comment:
We'll see - That was disabled in the same patch, I can still revert this - I just reverted the whole patch from pavel and fixed things that had to be